components/expenses/Orders.js

```diff
@@ -185,9 +216,56 @@ class Orders extends React.Component {
             </div>
           )}
         </div>
+        <Modal show={this.state.showModal} width="570px" onClose={() => this.setState({ showModal: false })}>
+          <ModalHeader>
+            <FormattedMessage id="order.cancel.modal.header" defaultMessage="Cancel Order" />
+          </ModalHeader>
+          <ModalBody>
+            <P>
+              <FormattedMessage
+                id="order.cancel.modal.body"
+                defaultMessage={'Are you sure you want to cancel this order?'}
+              />
+            </P>
+          </ModalBody>
+          <ModalFooter>
+            <Container display="flex" justifyContent="flex-end">
+              <StyledButton mx={20} onClick={() => this.setState({ showModal: false })}>
+                <FormattedMessage id="order.modal.cancel.btn" defaultMessage={'Cancel'} />
```

Comment:
[Not a requested change] Because we already use the word `Cancel` (`Are you sure you want to cancel this order?`) using it again for the call to action can be confusing - should I click on Cancel to Cancel my order? Maybe that's just me, but we could avoid this confusion with a simple `Yes` / `No` choice. (that would actually be good to have `id="yes" defaultMessage="Yes"` in our translations for other uses)